app/models/cdo/create/dog-details.js

```diff
@@ -0,0 +1,84 @@
+const { routes: ownerRoutes } = require('../../../constants/owner')
+const { routes, keys } = require('../../../constants/dog')
+
+function ViewModel (dogDetails, breedTypes, errors) {
+  this.model = {
+    formAction: routes.details.post,
+    backLink: ownerRoutes.confirmation.get,
+    breed: {
+      id: 'breed',
+      name: 'breed',
+      fieldset: {
+        legend: {
+          text: 'Breed type'
+        }
+      },
+      value: dogDetails[keys.breed],
+      items: breedTypes.map(type => ({
+        text: type.breed,
+        value: type.breed
```

Comment:
Should we not be using breed id as the value?

Reply:
I'd prefer to have the API do the mapping. Just gives us a bit more flexibility if we decide to change storage methods in the future.

Reply:
Ok, it's a bit of a dilemma since using id's gives other benefits. Happy to leave as is